edsdme/scripts/utils.js

```diff
@@ -50,7 +50,7 @@ export async function useMiloSample() {
 
 const miloLibs = setLibs('/libs');
 
-const { createTag, localizeLink, getConfig } = await import(`${miloLibs}/utils/utils.js`);
+const { createTag, localizeLink, getConfig, loadBlock } = await import(`${miloLibs}/utils/utils.js`);
```

Comment:
In performance pr this is removed from `utils.js`, so you can either import necessary MILO function in `scripts.js` and passed them as params to `getRenewBanner`, or import functions inside `getRenewBanner` function.